models/gameforuser.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+const {
+  Model
+} = require('sequelize');
+module.exports = (sequelize, DataTypes) => {
+  class GameForUser extends Model {
+    /**
+     * Helper method for defining associations.
+     * This method is not a part of Sequelize lifecycle.
+     * The `models/index` file will call this method automatically.
+     */
+    static associate(models) {
+      // define association here
+    }
+  }
+  GameForUser.init({
+    userId: DataTypes.INTEGER,
+    gameId: DataTypes.INTEGER,
+    playTime: DataTypes.FLOAT
+  }, {
+    sequelize,
+    modelName: 'GameForUser',
```

Comment:
I see you've changed the table names. Won't this affect the endpoints that are currently reading from a `usergame` table? Also, the field naming is inconsistent with the queries being done on the controller side. I'd advise you to make the necessary changes and make sure no controller fails, after running the migration.